src/schemas/lifecycle_management.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$id": "https://meta.comcast.com/firebolt/lifecycle_management",
+  "title": "LifecycleManagement",
+  "oneOf": [
+    {
+      "$ref": "#/definitions/State"
+    },
+    {
+      "$ref": "#/definitions/CloseReason"
+    }
+  ],
+  "definitions": {
+    "State": {
+      "type": "string",
```

Comment:
Due to the way we generate docs, we generally want a "title" attribute on shared schemas, otherwise they don't get their own doc sections.